Gopkg.lock

```diff
@@ -2,121 +2,89 @@
 
 
 [[projects]]
-  digest = "1:f1178b25c1ec9c61caf63e9a6b7a82c361fc0e8d7f98b68b831832a113476ba2"
```

Comment:
@darahayes @wtrocki Do the changes in this file look okay? GitHub hides hides the diff for it by default, so you may not have seen it (among the ~375 other hidden diffs).

I think it got updated when I ran `dep ensure` from `make setup`. I'm not sure why there would be changes as a result of that. Could it be down to my `go` or `dep` version or something?

```
$ go version
go version go1.11 linux/amd64

$ dep version
dep:
 version     : devel
 build date  : 
 git hash    : 
 go version  : go1.11beta2
 go compiler : gc
 platform    : linux/amd64
```

Reply:
Thanks @darahayes, I've done as you suggested and installed `dep` in the same way as the release step does it. That led to the changes in the Gopkg.lock going away, so I think everything is right with the world again! :+1: